client-libraries/0.1/how-to-guides/python/transfer-iota-tokens.md

```diff
@@ -53,75 +51,66 @@ To transfer your test tokens from one address to another, you need to create and
 
 1. Import the packages
 
-    ```go
-    package main
+    ```python
+    from iota import Iota
+    from iota import ProposedTransaction
+    from iota import Address
 
-    import (
-        . "github.com/iotaledger/iota.go/api"
-        "github.com/iotaledger/iota.go/trinary"
-        "github.com/iotaledger/iota.go/converter"
-        "github.com/iotaledger/iota.go/bundle"
-        "fmt"
-
-    )
     ```
 
-2. Connect to a node
+2. Define your seed. Replace this seed with one that owns an address with test IOTA tokens
```

Comment:
Link to how do I get testnet tokens for my seed?

Reply:
This is explained above in step 1